Makefile

```diff
@@ -236,6 +237,58 @@ check-pipeline-opa: ## Run policy evaluation for pipeline using opa. Deprecated.
 
 #--------------------------------------------------------------------
 
+##@ Bundles
+
+# Pushes two bundles, one for release and one for policy.
+# Each bundle includes policy/lib and its contents,
+# which is why we need the temp dir and the extra copying.
+# $(*) is expected to be either "release" or "pipeline".
+#
+.PHONY: push-bundles
+
+BUNDLE_REPO=quay.io/hacbs-contract
+BUNDLE_PREFIX=$(BUNDLE_REPO)/ec-policy
```

Comment:
I was considering just /policy here to make the repo name shorter, e.g. `https://quay.io/repository/hacbs-contract/ec-policy-release` vs `https://quay.io/repository/hacbs-contract/policy-release`.

Could also consider `https://quay.io/repository/hacbs-contract/release-policy` maybe for extra readability.